tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -1032,6 +1089,43 @@ def test_query_with_duplicates(self, retrieve_endpoints, one_mill_dps_ts, ms_bur
             assert len(res_lst.get(external_id=ts_numeric.external_id)) == 3
             assert len(res_lst.get(external_id=ts_string.external_id)) == 2
 
+    def test_retrieve_datapoints_in_target_unit(
```

Comment:
None of these tests uses `target_unit_system`, so I suggest you do a comparison between:
```
target_unit_system="Imperial"
# and
target_unit="temperature:deg_f"
```
and check for equality